vowpalwabbit/vw_predict.cc

```diff
@@ -0,0 +1,137 @@
+#include "vw_predict.h"
```

Comment:
Perhaps a name change inline with the binary.

Reply:
done